### PR TITLE
better yum header

### PR DESCRIPTION
**Pull request by evgeni**
- **document new candlepin repo location**
- **update foreman/katello version**
- **deploy yum header as a template**

### CODE REVIEW

File: puppet/modules/web/manifests/vhost/yum.pp [hunk 1]
```diff
@@ -63,14 +63,20 @@
     }
   }
 
-  ['HEADER.html', 'robots.txt', 'RPM-GPG-KEY-foreman'].each |$filename| {
```
Comment (evgeni) [inline puppet/modules/web/manifests/vhost/yum.pp:66]: I'd manually remove the file from web01, writing an absent section for one box seemed odd

File: puppet/modules/web/templates/yum/HEADER.html.epp
Comment (ekohl) [inline puppet/modules/web/templates/yum/HEADER.html.epp:1]: While you're at it, should we replace `latest` on line 16 with `$stable` so we start to move away from `latest`?

File: puppet/modules/web/templates/yum/HEADER.html.epp [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 <h1>yum.theforeman.org</h1>
 
 <h3>Foreman</h3>
 
 <p>Foreman is available under <code>/releases/VERSION/DIST/ARCH</code>, e.g.</p>
 
 <ul>
-  <li>/releases/3.8/el8/x86_64</li>
+  <li>/releases/<%= $stable %>/el8/x86_64</li>
   <li>/releases/nightly/el8/x86_64</li>
```
Comment (ekohl) [inline puppet/modules/web/templates/yum/HEADER.html.epp:10]: Should we also encourage el9?
```suggestion
  <li>/releases/<%= $stable %>/el9/x86_64</li>
  <li>/releases/nightly/el9/x86_64</li>
```

**Review by ekohl — APPROVED** [review]

File: puppet/modules/web/templates/yum/HEADER.html.epp (hunk 2 above)
Comment (evgeni) [reply puppet/modules/web/templates/yum/HEADER.html.epp]: yes, did a `s/el8/el9/g` on the whole file

File: puppet/modules/web/templates/yum/HEADER.html.epp
Comment (evgeni) [reply puppet/modules/web/templates/yum/HEADER.html.epp:1]: yes, done

**Pull request merged**